components/brave_ads/core/internal/common/task/task_util.cc

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_ads/core/internal/common/task/task_util.h"
+
+#include "base/functional/callback.h"
+#include "base/task/sequenced_task_runner.h"
+#include "base/time/time.h"
+
+namespace brave_ads {
+
+void PostDelayTaskToCurrentSequence(base::OnceClosure task,
+                                    base::TimeDelta delay) {
+  base::SequencedTaskRunner::GetCurrentDefault()->PostDelayedTask(
+      FROM_HERE, std::move(task), delay);
```

Comment:
I am in the opinion we should just call this directly and remove the util

Reply:
fixed